### PR TITLE
nit-picking the examples

### LINKED ISSUE #12: Nit-picking the examples
Please allow me to make some minor changes in the examples (using the new MeshModifiers, dip_angle=0 for P-sample).

### PR DESCRIPTION
**Pull request by jmeier**
Please allow me to make some minor changes in the examples (using the new MeshModifiers, dip_angle=0 for P-sample).
Closes #12

### CODE REVIEW

File: examples/lab_testing_CU/lab_test_P_sample.i [hunk 1]
```diff
@@ -4,7 +4,7 @@
 pconf = 8 #MPa --> the initial effective confining pressure applied to the sample before shearing
 pw = 9 # the initial pore pressure after consilidation phase
 pconf_total = ${pconf}+${pw} #MPa Total confining presure
-dip_angle = 90
```
Comment (Kavan-Khaledi) [inline examples/lab_testing_CU/lab_test_P_sample.i:7]: dip_angle=90 for P sample is correct. But, for S-Sample it should be 0, which is wrong in S sample example. Could you please correct it?

Comment (jmeier) [reply examples/lab_testing_CU/lab_test_P_sample.i]: Sure, done.

**Review by Kavan-Khaledi — CHANGES_REQUESTED** [review]
Could you please change the dip_angle of S example to 0, and keep the dip_angle of P sample equal to 90

**Comment by jmeier** [comment]
> Could you please change the dip_angle of S example to 0, and keep the dip_angle of P sample equal to 90

Please check my last amend: this is now the case.

**Pull request merged**